src/docs/arch-hs.dox

```diff
@@ -1,8 +1,115 @@
 /*! @arch_page arch-hs History Store
 
-The History Store in WiredTiger tracks old (all but the latest committed)
+The history store in WiredTiger tracks old (all but the latest committed)
 versions of records.  By having these records in storage separate from
```

Comment:
"  By having" I think there is an extra space before "By"

Reply:
Fixed.